internal/provider/service_accounts_datasource_test.go

```diff
@@ -0,0 +1,31 @@
+package provider
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+)
+
+func TestAccServiceAccounts(t *testing.T) {
+	resource.Test(t, resource.TestCase{
```

Comment:
Another test that first adds a service account, or two and then makes sure they show up in the data source could be useful